nautobot/extras/tests/test_customfields.py

```diff
@@ -568,6 +568,21 @@ def test_minimum_maximum_values_validation(self):
         response = self.client.patch(url, data, format="json", **self.header)
         self.assertHttpStatus(response, status.HTTP_200_OK)
 
+    def test_bigint_values_of_custom_field_maximum_attribute(self):
```

Comment:
Thank you for including test coverage! Would you mind adding a similar basic test case to cover bigint values for `validation_minimum` as well?

Reply:
I will include a test to cover bigint values for `validation_minimum` as well.

Reply:
https://github.com/Joezeppe/nautobot/commit/96d97e5c9dd6d4211ba5a258e2a6846e672d8cbc#diff-8bd6d9808e1f0ae1ceb512f17c5431c2c2ab9ed78661648d683a2952bb1b0423R583